routes/route.js

```
@@ -1,15 +1,46 @@
+  return 'I am not used';
+}
+
+// Duplicate function (Duplicate/Unused Code)
+function unusedFunc() {
+  return 'Still not used';
+}
+
+// Large, unmodular function (Code Structure)
+async function bigRouteHandler(req, res) {
+  let a = 1, b = 2, c = 3, d = 4, e = 5, f = 6, g = 7, h = 8, i = 9, j = 10;
+  if (a = 2) { // Logic error: should be == or ===
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)